package.json

```diff
@@ -50,7 +50,8 @@
     "tests:unit:debug": "DEBUG=true ./test/dockerized-unit-tests.sh",
     "tests:integration": "./test/dockerized-integration-tests.sh",
     "tests:integration:debug": "DEBUG=true ./test/dockerized-integration-tests.sh",
-    "compile": "webpack",
+    "compile-noeslint": "webpack --mode=development",
```

Comment:
@laemtl can we make `compile` be the development mode and `compile-eslint` or `compile-production` be for production?

Reply:
Sure. Travis is running run-js-linter.sh I think, not the npm command, is that right?

Reply:
I'm unsure about Travis but Makefile uses `npm run compile`

Reply:
@kongtiaowang might know.

Reply:
@laemtl also docs/React.README.md mentions npm run compile

Reply:
I checked, Travis uses npm run lint:javascript. So do we need a compile-production? A combo of compile (with eslint off) and npm run lint:javascript can achieve the same result.

Reply:
I just would like to keep `npm run compile` for development since it's simpler to type. I think Makefile is used when setting up production VMs.